bugwarrior/services/logseq.py

```diff
@@ -0,0 +1,301 @@
+import logging
+
+import requests
+import typing_extensions
+
+import re
+from datetime import datetime
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class LogseqConfig(config.ServiceConfig):
+    service: typing_extensions.Literal["logseq"]
+    host: str = "localhost"
+    port: int = 12315
+    token: str = ""
```

Comment:
```suggestion
    token: str
```

Leave out the default value in order to make this required. I'd do the same with the `port` too unless 12315 is Logseq's default value.